test/EzMap.IntegrationTest/Infrastructure/HttpHelper.cs

```diff
@@ -5,7 +5,8 @@ namespace EzMap.IntegrationTest.Infrastructure;
 
 public static class HttpHelper
 {
-    public static Task<HttpResponseMessage> RequestAsJsonAsyncWithToken<T>(this HttpClient client, HttpMethod method,
+    public static Task<HttpResponseMessage> 
```

Comment:
Async should be suffixed, not in the middle of the method's name. It's worse than not having Async at all.